enhancements/operator-conditions.md

```diff
@@ -64,13 +64,18 @@ As an operator author, I want operators managed by OLM to be able to communicate
 
 ## Design Details
 
-### Condition CRD
+### status.operators.coreos.com CRD
 
-OLM will create a channel for operators to communicate complex conditions by introducing the `Condition` [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+OLM will create a channel for operators to communicate complex conditions by introducing the `Status` [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
 
-When an operator is installed, OLM will create a `Condition` CustomResource (CR) for the operator. The operator should only be concerned with updating the status of the `Condition` that it owns and will have its RBAC scoped so it may only GET, LIST, or UPDATE the `Condition` CR it is associated with.
+When an operator is installed, OLM will create a `Status` CustomResource (CR) for the operator. The operator's RBAC will be configured such that it is only able to get the `Status` resource it own and update the resources [status subresource](https://book-v1.book.kubebuilder.io/basics/status_subresource.html).
 
-The `Condition` CR will be created with the [aggregation label](https://github.com/openshift/enhancements/blob/master/enhancements/olm/simplify-apis.md#component-selection) introduced for the `Operator` CRD that will associate it with a specific operator. When OLM deploys the operator, an additional environment variable will be appended to the deployment. Operators will then be able to use this label along with a library provided by the [operator-sdk](https://github.com/operator-framework/operator-sdk) team to set [conditions](https://github.com/kubernetes/enhancements/blob/3aa92e20bdfa2e60e3cb1a2b92cdca61847d7ad2/keps/sig-api-machinery/1623-standardize-conditions/README.md#kep-1623-standardize-conditions) on the CR to communicate complex states to OLM. For those unfamiliar with conditions, they are used by many Kubernetes Resources to provide users with additional insight into the state of the resource and typically adhere to the following format:
+When the `Status` CR is created, OLM will update the deployments defined in the CSV with two addition environment variables:
+
+- *OperatorNamespace*: The namespace that the operator is running in
```

Comment:
I don't think OLM needs to add this, since it's a standard use case for the [downward API](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-pod-fields-as-values-for-environment-variables).

Reply:
Pods are not automatically generated with this information - are you suggesting that we instruct the sdk to always include the field reference?

Reply:
If so - that seems fine to me.

CC @varshaprasad96 @joelanford please comment if you object.

Reply:
> are you suggesting that we instruct the sdk to always include the field reference?

No, just that the operator should know what namespace it's in, one way or another -- that environment variable can be added manually.